.github/workflows/integration-tests.yml

```diff
@@ -89,17 +90,45 @@ jobs:
       - name: Run integration tests
         run: |
           cd tests
-          tmt run -v -eCONTAINER_USED=integration-test-local
+          tmt run -v -eCONTAINER_USED=integration-test-local -eWITH_COVERAGE=1
 
       - name: Show tmt log output in case of failure
         if: ${{ failure() }}
         run: |
           cat /var/tmp/tmt/run-001/log.txt
 
-      - name: Gather tests artifacts
+      # Extract the collected coverage results if they exist and remove them from
+      # the tmt directory to avoid duplication in artifacts
+      - name: Extract coverage results
+        if: always()
```

Comment:
Even if the tests fail, the coverage might provide valuable insights about which code parts were executed. This is a bit limited, though, since we only provide merged coverage info for all cases.

Reply:
OK